docs/snippets/all/concepts/different_data_per_timeline.cpp

```diff
@@ -0,0 +1,24 @@
+// Log different data on different timelines.
+
+#include <rerun.hpp>
+
+int main() {
+    const auto rec = rerun::RecordingStream("rerun_example_different_data_per_timeline");
+    rec.spawn().exit_on_failure();
+
+    rec.set_time_sequence("blue timeline", 0);
+    rec.set_time_seconds("red timeline", 0.0);
```

Comment:
nit: inconsistent int vs float throughout

Reply:
that was intentional just to illustrate that we can mix sequence & seconds 🤷 
(and sequence are always int and seconds are always float)

Reply:
Yup, dont mind me, I can't read 🤦🏻